verification/verify.go

```diff
@@ -110,9 +119,7 @@ func (sv *SegmentVerifier) Verify(params *Params) (*Params, error) {
 	// Check pixel counts
 	if (err == nil || IsRetryable(err)) && res != nil && params.Results != nil {
 		if len(res.Pixels) != len(params.Results.Segments) {
-			// TODO make allowances for the verification algos not doing
-			//      pixel counts themselves; adapt broadcast.go verifyPixels
-			err = ErrPixelsAbsent
+			err = verifyPixelParams(params)
```

Comment:
An additional thought about the approach of using a default implementation of the `Verifier` interface - that implementation actually wouldn't even need to do the pixel counting if we're also accommodating for verifiers that don't do pixel counting here.